cmd/atlasci/main.go

```diff
@@ -29,6 +29,10 @@ import (
 func main() {
 	var opt options
 	check(opt.parse())
+	if opt.license {
+		fmt.Println(license)
+		return
+	}
```

Comment:
You can avoid this field and just call `os.Exit(0)` in parse.

Reply:
done